spec.md

```diff
@@ -752,7 +757,11 @@ A Controller Plugin MUST implement this RPC call if it has `CREATE_DELETE_VOLUME
 This RPC will be called by the CO to provision a new volume on behalf of a user (to be consumed as either a block device or a mounted filesystem).
 
 This operation MUST be idempotent.
-If a volume corresponding to the specified volume `name` already exists, is accessible from `accessibility_requirements`, and is compatible with the specified `capacity_range`, `volume_capabilities` and `parameters` in the `CreateVolumeRequest`, the Plugin MUST reply `0 OK` with the corresponding `CreateVolumeResponse`.
+If a volume corresponding to the specified volume `name` already exists, is accessible from `accessibility_requirements`, and is compatible with the specified `capacity_range`, `volume_capabilities`, `parameters` and `mutable_parameters` in the `CreateVolumeRequest`, the Plugin MUST reply `0 OK` with the corresponding `CreateVolumeResponse`.
+
+The `parameters` field SHALL contain opaque volume attributes to be specified at creation time. 
+The `mutable_parameters` field SHALL contain opaque volume attributes attributes that are defined at creation time but MAY also to be changed during the lifetime of the volume via a subsequent `ModifyVolume` RPC. 
+Values specified in `mutable_parameters` SHOULD take precedence over the values from `parameters`.
```

Comment:
any reason in particular the wording is SHOULD vs MUST?

Reply:
There is no way to enforce MUST?

Reply:
that doesn't stop us from using SHALL or MUST elsewhere...

Reply:
@bswartz How strong do you think our wording should be around this?

Reply:
The only argument for SHOULD is that both fields are opaque and it's really hard to know what will go on inside the SP. That being said, I can't think of a specific example where an SP might NEED to violate this rule so I'm not opposed to using MUST instead. MUST definitely makes the expected semantic clearer on the CO side.

Reply:
Use MUST instead of SHOULD